spartan/aztec-validator/values.yaml

```diff
@@ -14,6 +14,8 @@ validator:
   mnemonic: "test test test test test test test test test test test junk"
   mnemonicStartIndex: 2000
   validatorsPerNode: 1
+  publishersPerValidatorKey: 2
+  publisherMnemonicStartIndex: 5000
```

Comment:
Same as previous message but this set should be disjoint from the set used in the prover